src/main/java/io/tiledb/spark/Range.java

```diff
@@ -18,7 +18,13 @@
 
   Range(Pair range) {
     this.range = range;
-    this.dataClassType = range.getFirst().getClass();
+    if (range.getFirst() != null) this.dataClassType = range.getFirst().getClass();
+    else if (range.getSecond() != null) this.dataClassType = range.getSecond().getClass();
```

Comment:
Missing `else` handling?

Reply:
No need to handle that case. The classType will be null.

Reply:
Thanks